.travis.yml

```diff
@@ -4,11 +4,11 @@ cache: pip
 services:
   - docker
 env:
-  - ANSIBLE=2.2.3
-  - ANSIBLE=2.3.3
-  - ANSIBLE=2.4.3
+  - ANSIBLE='ansible>=2.2.0,<2.3.0'
+  - ANSIBLE='ansible>=2.3.0,<2.4.0'
+  - ANSIBLE='ansible>=2.4.0,<2.5.0'
 install:
-  - pip install ansible==${ANSIBLE} ansible-lint>=3.4.15 molecule==1.25.0 docker git-semver 'testinfra>=1.7.0,<=1.10.1'
+  - pip install ${ANSIBLE} 'ansible-lint>=3.4.15' 'molecule==1.25.0' docker git-semver 'testinfra>=1.7.0,<=1.10.1'
```

Comment:
An alternative way to do this is:

```
ANSIBLE='>=2.2.0,<2.3.0'
pip install ansible${ANSIBLE}
```

But I think the approach taken in the PR is more readable.

Reply:
I agree, one from PR is better.